Fix find_reaching_copies for asm call with @PLT

romainducrocq opened the pull request:
On Linux, the copy propagation tester doesn't find asm `call` instruction when using the suffix <name>@PLT.
Fix: If the asm call is not found on Linux, append @PLT to the first arg (function name) and try again.

nlsandler commented:
Thanks for opening this PR, and good catch! I'd actually suggest fixing this differently: when we parse a `call` instruction in parse.py, just strip the @PLT suffix from the target, if there is one. I think the logic will be a little simpler, and it's also more general (I think we want to ignore the @PLT suffix pretty much all the time, not just when testing copy prop).

Does that sound like a good solution to you? And would you like to implement it or just have me handle it?

Btw, if you do want to implement this fix, you can use something like
```
if operand.lower().endswith("@plt"):
    # strip the suffix
```

so you don't need to spell out every case variation of `plt`

romainducrocq commented:
Your solution is obviously better (i did the patch for my own convenience without thinking too much about it). I updated the PR and squashed the changes, thanks!

nlsandler commented:
Looks good, thank you!
